ccip-sdk/src/chain.ts

```diff
@@ -151,6 +151,38 @@ export type UnsignedTx = {
   [ChainFamily.Sui]: never // TODO
 }
 
+/**
+ * Common options for [[generateUnsignedSendMessage]] and [[sendMessage]] Chain methods
+ */
+export type SendMessageOpts = {
```

Comment:
Are we reexporting SendMessageOpts and ExecuteReportOpts from index?

Reply:
Nope, because I said, I don't want them to be used/referred directly, and instead the actual declarations to be the reference point